src/internal/interfaces/AxisType.ts

```diff
@@ -1,6 +1,11 @@
 // (C) 2019 GoodData Corporation
 export type AxisType = "xaxis" | "yaxis" | "secondary_xaxis" | "secondary_yaxis";
 
+export interface IAxisNameProperties {
+    visible?: boolean;
+    position?: string;
```

Comment:
dont we want to reuse Highcharts.AxisTitleAlignValue type or create our own alias for it and use it instead of generic string?

Reply:
I want to keep it as generic string.
Because its value could be `auto | left | center | right` | bottom | middle |top` for AD and then transformed to `low | middle | high` for SDK.